static/js/font.js

```diff
@@ -0,0 +1,19 @@
+function updateFontVariations() {
```

Comment:
direct rip of [live site](https://github.com/canonical/design.ubuntu.com/blob/main/static/js/font.js)